docs/internals/RFC-feature-coverage.md

```diff
@@ -0,0 +1,168 @@
+## TL;DR.
+
+We'll implement a mechanism that will put metadata on each test class about the coverage/completness of the test. Two questions : 
+
+* There will be three level, `basic`, `good`, `complete`. It's a compromise, do you think it's too many? not enough?
```

Comment:
We could consider these as enums for something like a score? What if we allowed a max score of 100 and did something like allow a manual specification of the score if these are not precise enough?
`Basic = 30, Good = 60, Complete = 100`

That said, I think this will be granular enough for most cases anyways.